### PR TITLE
font update

### PR DESCRIPTION
**Pull request by Arhell**
[issue ](https://github.com/zephir-lang/zephir-docs/issues/351)

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .hljs{display:block;overflow-x:auto;padding:1em;background:#2b2b2b}.hljs{color:#bababa}.hljs-strong,.hljs-emphasis{color:#a8a8a2}.hljs-bullet,.hljs-quote,.hljs-link,.hljs-number,.hljs-regexp,.hljs-literal{color:#6896ba}.hljs-code,.hljs-selector-class{color:#a6e22e}.hljs-emphasis{font-style:italic}.hljs-keyword,.hljs-selector-tag,.hljs-section,.hljs-attribute,.hljs-name,.hljs-variable{color:#cb7832}.hljs-params{color:#b9b9b9}.hljs-string{color:#6a8759}.hljs-subst,.hljs-type,.hljs-built_in,.hljs-builtin-name,.hljs-symbol,.hljs-selector-id,.hljs-selector-attr,.hljs-selector-pseudo,.hljs-template-tag,.hljs-template-variable,.hljs-addition{color:#e0c46c}.hljs-comment,.hljs-deletion,.hljs-meta{color:#7f7f7f}
 
 body {
-    font-family: Merriweather, "EB Garamond", Georgia, serifbody;
+    font-family: 'Open Sans', sans-serif;
```
Comment (sergeyklay) [inline assets/css/style.css:4]: @Arhell  How about those who don't have installed font?

Comment (Arhell) [reply assets/css/style.css:4]: I add to head link and fallback to sans-serif or you mean 
font-family: 'Open Sans', sans-serif, Merriweather, "EB Garamond", Georgia, serifbody; ?

Comment (sergeyklay) [reply assets/css/style.css:4]: Fine! Do we still need `https://fonts.googleapis.com/css?family=Merriweather` in the header?

Comment (Arhell) [reply assets/css/style.css:4]: Updated

**Comment by sergeyklay** [comment]
Thank you!

**Pull request merged**